Lib/_pydatetime.py

```diff
@@ -215,6 +215,17 @@ def _need_normalize_century():
             _normalize_century = True
     return _normalize_century
 
+_supports_c99 = None
+def _can_support_c99():
+    global _supports_c99
+    if _supports_c99 is None:
+        try:
+            _supports_c99 = (
+                _time.strftime("%F", (1900, 1, 1, 0, 0, 0, 0, 1, 0)) == "1900-01-01")
+        except ValueError:
+            pass
```

Comment:
Should not `_supports_c99` be set to False here?

Reply:
Ah well spotted. Thanks!